intro/index.html

```diff
@@ -0,0 +1,163 @@
+                            <img width="500" data-src="../coders_school_logo.png" alt="Coders School" class="plain">
+                        </a>
+                        <h3>Mateusz Adamski</h3>
+                        <h3>Łukasz Ziobroń</h3>
+
+                    </section>
+                    <section data-markdown>
+                        <textarea data-template>
+                        <!-- .slide: style="font-size: 0.9em" -->
+
+                        ## Agenda
+
+                        * <!-- .element: class="fragment fade-in" --> Typy danych
+                        * <!-- .element: class="fragment fade-in" --> Funkcje
+                        * <!-- .element: class="fragment fade-in" --> Instrukcje warunkowe
+                        * <!-- .element: class="fragment fade-in" --> Pętle
+                        * <!-- .element: class="fragment fade-in" --> Tablice
+                        * <!-- .element: class="fragment fade-in" --> STL - co to?
+                        * <!-- .element: class="fragment fade-in" --> <code>std::vector</code>
+                        * <!-- .element: class="fragment fade-in" --> Pętla <code>for</code> po kolekcji
+                        * <!-- .element: class="fragment fade-in" --> <code>std::string</code>
+                        * <!-- .element: class="fragment fade-in" --> <code>std::list</code>
+                        * <!-- .element: class="fragment fade-in" --> <code>std::map</code>
+                        * <!-- .element: class="fragment fade-in" --> Zakresy (scopes)
+                        * <!-- .element: class="fragment fade-in" --> Referencje
+                        * <!-- .element: class="fragment fade-in" --> Wskaźniki
+                        * <!-- .element: class="fragment fade-in" --> Zagrożenia przy stosowaniu referencji i wskaźników
+                        * <!-- .element: class="fragment fade-in" --> Typ wyliczeniowy <code>enum</code> i <code>enum class</code>
+
+                        </textarea>
+                    </section>
+                    <section data-markdown>
+                        <textarea data-template>
+
+                        ## Zadania
+
+                        [Repo GH `coders-school/cpp-fundamentals`](https://github.com/coders-school/cpp-fundamentals/tree/master/module1/)
+
+                        </textarea>
+                    </section>
+                </section>
+                
+                <section>
+                    <section data-markdown="about_you.md" data-separator-vertical="^___"
+                        data-separator-notes="^Note:">
+                    </section>
+
+                    <section data-markdown="about_lukasz.md" data-separator-vertical="^___"
+                        data-separator-notes="^Note:">
+                    </section>
+
+                    <section data-markdown="contract.md" data-separator-vertical="^___"
+                        data-separator-notes="^Note:">
+                    </section>
```

Comment:
Zróbmy może tak. Te pliki wspólne – about_you.md, about_lukasz.md i contract.md oraz img/lukasz.jpg wrzućmy do repo coders-school/reveal.js i podlinkujmy tutaj (tak jak np css w linii 14 jest z tamtego repo linkowany - ../../css/theme/coders.css). Wtedy będzie jeszcze mniej duplikacji pomiędzy wszystkimi repozytoriami. W tych repo, w których już wydzielałeś intro też by to trzeba było tak podlinkować.